pixeltable-mcp/base-sdk/tools.py

```
@@ -1,9 +1,126 @@
+    if isinstance(node, ast.BinOp):
+        op_func = _SAFE_OPERATORS.get(type(node.op))
+        if op_func is None:
+            raise ValueError(f"Unsupported operator: {type(node.op).__name__}")
+        left = _eval_node(node.left, namespace)
+        right = _eval_node(node.right, namespace)
+        return op_func(left, right)
+
+    # Comparison operations
+    if isinstance(node, ast.Compare):
+        left = _eval_node(node.left, namespace)
+        for op, comparator in zip(node.ops, node.comparators):
```

Ruff (0.15.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)